libs/wait-for-ecs-deploy/src/__mocks__/ecs-describe-services-all-services-deployed.json

```diff
@@ -0,0 +1,606 @@
+{
+    "services": [
```

Comment:
This looks super brittle? Is it supposed to be checked in

Reply:
This is the mocked response from the ECS API; could probably replace it with dummy values if needed.

Reply:
Discussed offline
* added README that describes how the files were created